src/views/Subrecipient.vue

```diff
@@ -1,93 +1,145 @@
 <template>
   <div>
-    <h1>Subrecipient</h1>
-    <div v-if="loading">
-      Loading..
+    <h2>Subrecipient {{ recipientId }}</h2>
+
+    <div v-if="!recipient" class="spinner-grow text-primary" role="status">
+        <span class="sr-only">Loading...</span>
     </div>
+
     <div v-else>
-      <RecordForm
-        type="Subrecipients"
-        :columns="fields"
-        :record="editSubrecipient"
-        :id="editSubrecipient.id"
-        :isNew="isNew"
-        :onSave="onSave"
-        :onCancel="onCancel"
-        :onDone="onDone"
-        :errorMessage="errorMessage"
-      />
+      <div class="form-group row">
+        <div class="col-sm-2">
+          Created:
+        </div>
+        <div class="col-sm-10">
+          In
+          <router-link :to="`/uploads/${recipient.upload_id}`">
+            Upload {{ recipient.upload_id }}
+          </router-link>
+          on {{ humanDate(recipient.created_at) }}
+          by {{ recipient.created_by }}
+        </div>
+      </div>
+
+      <div class="form-group row">
+        <div class="col-sm-2">
+          Updated:
+        </div>
+        <div class="col-sm-10">
+          <span v-if="!recipient.updated_at">Never manually updated</span>
+          <span v-else>
+            By {{ recipient.updated_by_email }} on {{ humanDate(recipient.updated_at) }}
+          </span>
+        </div>
+      </div>
+
+      <div class="form-group row" v-for="(rule, key) in rules" :key="key">
+        <label :for="key" class="col-sm-2 col-form-label">{{ rule.humanColName }}</label>
+        <div class="col-sm-10">
+          <select v-if="rule.listVals.length > 0" :id="key" v-model="record[key]" :readonly="isReadOnly(key)">
```

Comment:
```suggestion
          <select v-if="rule.listVals.length > 0" :id="key" v-model="record[key]" :readonly="isReadOnly(key)" class="form-control">
```

This should have the `form-control` class also.